tsdb/index/index.go

```diff
@@ -1505,6 +1510,26 @@ func (r *Reader) LabelValues(name string) ([]string, error) {
 	return values, nil
 }
 
+// LabelValueFor returns value of given label value of series referred to by ID.
```

Comment:
Same comment modifications as suggested somewhere else

Reply:
thx, fixed in https://github.com/prometheus/prometheus/pull/8400/commits/c2f560728cbec5ea14c276fa2f1ca06c8232e079